TextToSpeech/TextToSpeechImplementation.cpp

```diff
@@ -326,7 +406,11 @@ namespace Plugin {
     {
         CHECK_TTS_MANAGER_RETURN_ON_FAIL();
         _adminLock.Lock();
-        auto status = _ttsManager->shut(speechid);
+        auto status = TTS::TTS_FAIL;
```

Comment:
May I know why speechid is not validated on Pause() & Resume() operations?

Reply:
if we pass invalid speechid  also it will equals with  currentspeechid which is speaking, Error will be thrown at that stage if speechid != currentspeechid